### DISCUSSION TITLE
如何绕过代理软件？

### DISCUSSION THREAD

**Discussion by UltraUser** [NewFuture/DDNS#309] [Q&A]
开了某猫型图标软件后，域名解析记录会指向代理服务器的IP……by pass 规则貌似不起作用

**Comment by NewFuture** [comment]
默认是出口IP，内外IP可以取网卡或者自定义命令

https://github.com/NewFuture/DDNS#index4-%E5%92%8C-index6-%E5%8F%82%E6%95%B0%E8%AF%B4%E6%98%8E

外网 public

**Reply by UltraUser** [reply]
> 默认是出口IP，内外IP可以去网卡或者自定义命令
> 
> https://github.com/NewFuture/DDNS#index4-%E5%92%8C-index6-%E5%8F%82%E6%95%B0%E8%AF%B4%E6%98%8E

谢谢，用pbulic就解决了……一开始想多了，用的url，反而不行
